docs/source/intro.rst

```diff
@@ -180,42 +184,87 @@ The simplest form is::
 
 	notepad.clearCallbacks()
 
-This unregisters all callbacks for all events.  If you want to just clear one or more events, just pass the list of :class:`NOTIFICATION` events you wish to clear.::
+This unregisters all callbacks for all new events.  If you want to just clear one or more events, just pass the list of :class:`NOTIFICATION` events you wish to clear.::
 
 	notepad.clearCallbacks([NOTIFICATION.FILEBEFORESAVE, NOTIFICATION.FILESAVED])
 
 *Note that if you want to clear the callback for just one event, you still need to pass a list (i.e. surrounded with square brackets)*
 
-To unregister a callback for a particular function, just pass the function.::
+To unregister all callback for a particular function, just pass the function::

```

Comment:
A callback seems to be more accurate because Notepad has no asynchronous callbacks, which means that there is always one callback to this function at any time.

Reply:
Only one callback will be at work/active at a particular time, yes. 
But there may be several callbacks registered: for same function, but for different event types. 
Like in: 
```py
notepad.callback(myfun, [NOTIFICATION.EVENTTYPE1])
notepad.callback(myfun, [NOTIFICATION.EVENTTYPE2])

# notepad.clearCallbacks(myfun,  [NOTIFICATION.EVENTTYPE1] ) # would only clear the callback for this event type ,1
notepad.callback(myfun)   # will clear both callbacks 
```
Also, it's how it's defined in /docs/source/notepad.rst : 
> notepad.clearCallbacks(function)
Unregisters all callbacks for the given function.

Reply:
Or do we say instead that in lines 1 & 2 above only 1 callback has been registered ?

Reply:
To be honest I never thought about it but you are right, is that two recalls now or still one? @chcg what do you think? And if we differentiate it between "all callbacks" for the editor object and just "a callback" for the notepad object, that confuses people too.